vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -19607,6 +19666,23 @@ unit:MilliJ
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "Millijoule"@en ;
 .
+unit:MilliJ-PER-GM
+  a qudt:Unit ;
+  dcterms:description "The 0.001-fold of the SI base unit joule divided by the 0.001-fold of the SI base unit kilogram."^^rdf:HTML ;
+  qudt:applicableSystem sou:CGS ;
+  qudt:applicableSystem sou:CGS-EMU ;
+  qudt:applicableSystem sou:CGS-GAUSS ;
+  qudt:applicableSystem sou:SI ;
+  qudt:conversionMultiplier 1000000000.0 ;
```

Comment:
Multiply top and bottom by 1000 and you get J-PER-KiloGM, so the conversionMultiplier must be 1.0

Reply:
How did I manage to do that? 🤦‍♂️ Fixed.